### PR TITLE
Enables 'tables' extension in python-markdown initialization

### LINKED ISSUE #1486: bug/Tables are not being parsed from markdown files
**Describe the bug**
Tables are being returned as UncategorizedText . 

**To Reproduce**

Example Markdown file:
```

# A table

Item | Price | # In stock
---|---|---
Juicy Apples | 1.99 | 739
Bananas | 1.89 | 6

```

**Expected behavior**
It should return a  Table element.

**Additional context**
The reason for this is because for python-markdown to support tables, it needs to have an extension enabled: https://python-markdown.github.io/extensions/tables/

Changing this:
`html = markdown.markdown(text)`
to this:
`html = markdown.markdown(text, extensions=['tables'])`
in partition_md, fixes the issue.

I can submit a pull request if that's an acceptable solution.

### PR DESCRIPTION
**Pull request by phowat**
Enables extension to correctly parse markdown tables.
Fixes #1486

### CODE REVIEW

**Comment by qued** [comment]
Could you add a test, maybe something that processes the example markdown file you provided and verifies that a `Table` element is extracted? Also add an entry in `CHANGELOG.md` under "Fixes" and bump the version there and in `unstructured/__version__.py`.

**Comment by phowat** [comment]
Hi, added the test and changelog info.

File: unstructured/partition/md.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def partition_md(
 
         text = response.text
 
-    html = markdown.markdown(text)
+    html = markdown.markdown(text, extensions=['tables'])
```
Comment (badGarnet) [inline unstructured/partition/md.py:89]: does this require `markdown.extensions` package?

Comment (cragwolfe) [reply unstructured/partition/md.py]: apparently not

**Review by cragwolfe — APPROVED** [review]
thanks for the contribution, @phowat !

**Comment by qued** [comment]
Merged near-duplicate PR [here](https://github.com/Unstructured-IO/unstructured/pull/1924) that was based on this one. Thanks @phowat!

**Pull request closed without merging**